dockerizeit/master/jenkins.properties

```diff
@@ -59,6 +59,21 @@ slack {
   }
 }
 
+bitbucketEndpoints {
+   bitbucket_server {
+      type = "server"
```

Comment:
that's a nice improvement!
I'd like to suggest small change: in most of possible configurations we introduced 'enable' field, like in

```
gerrit {
  example {
    enabled = false
    hostName = "gerrit.example.com"
    sshPort = 29418
    frontendURL = "http://gerrit.example.com:8081/"
    proxy = "http://proxy:8080"
    userName = "user"
    sshKeyFile = "/var/jenkins_home/.ssh/id_rsa"
    email = "user@example.com"
  }
}
```
and then you check in the script if it's actually enabled, before setting it up. It's an easy way to disable selected configuration without removing it completely from the file, so it's easy to restore it back. 
What do you think?